tokio/src/runtime/builder.rs

```diff
@@ -198,7 +198,12 @@ impl Builder {
     /// // This will run the runtime and future on the current thread
     /// rt.block_on(async move {});
     /// ```
+    ///
+    /// # Panic
+    ///
+    /// This will panic if `val` is not larger than `0`.
     pub fn worker_threads(&mut self, val: usize) -> &mut Self {
+        assert!(val > 0, "Worker threads can not be set to 0");
```

Comment:
I'm unsure about this one, but I feel like it should be without a space?
```suggestion
        assert!(val > 0, "Worker threads cannot be set to 0");
```